src/content/eventList/eventList.ts

```diff
@@ -0,0 +1,102 @@
+import type { SparcsEvent } from "../../models/SparcsEvent.astro";
+
+export interface TechTixEventData {
+  eventSource: "TechTix";
+  eventId: string;
+  name: string;
+  bannerUrl: string;
+  startDate: string | Date;
+  endDate: string | Date;
+  description: string | null;
+}
+
+const TECHTIX_API =
+  "https://ru009rx9sk.execute-api.ap-southeast-1.amazonaws.com/dev/events";
```

Comment:
Store the base url in an environment variable named `TECHTIX_API`. We cannot hardcode this because the link may change depending on the environment (i.e. dev or prod). 

```
const TECHTIX_API = import.meta.env.TECHTIX_API + '/events'
```

Learn more here:
https://docs.astro.build/en/guides/environment-variables/